assets/sass/_docs.sass

```diff
@@ -0,0 +1,47 @@
+/* Header & Hero */
+.single-hero .hero-body
+  padding: 1.5rem
+  code
+    border-radius: 2px
+    padding: 0.1rem 0.2rem 0.1rem 0.2rem
+
+.title
+  @extend .title
+  @extend .is-1
+  font-weight: 200
+  margin-top: 1rem
+  margin-bottom: 1.5rem
+
+.subtitle
+  @extend .subtitle
+  color: #a3a3a3
+
+/* Left Nav */
+#TableOfContents
+  ul
+    @extend .menu-list
+  a
+    border-bottom: 1px solid $cncf-light-gray
+  li ul, ul li ul
+    margin: 0
+
+/* Body */
+.hashlink
+  @extend .icon
+  font-size: 1rem
+  height: 1rem
+  width: 1rem
+
+.headline-hash
+  display: none
+
+h1:hover > .headline-hash, h2:hover > .headline-hash, h3:hover > .headline-hash,
```

Comment:
Bonus points if you can clean this up with a [Sass @for loop](https://sass-lang.com/documentation/at-rules/control/for).